packages/components/addon/components/hds/form/select/base.hbs

```diff
@@ -0,0 +1,3 @@
+<select class={{this.classNames}} {{style width=@width}} ...attributes>
+  {{yield (hash Options=(component "hds/yield"))}}
```

Comment:
It's worth noting that this being a generic yield it will accommodate any mix of `<optgroup>` and `<options>`, making it more flexible.

Reply:
That's a **very** good point! I've updated the documentation to include also this option (pun intended!): https://github.com/hashicorp/design-system/pull/397/commits/025075f76e1389c5aa5feaeb0b2b90676ae1e35d